stdlib/3/collections/__init__.pyi

```diff
@@ -59,9 +59,133 @@ else:
     def namedtuple(typename: str, field_names: Union[str, Iterable[str]],
                    verbose: bool = ..., rename: bool = ...) -> Type[tuple]: ...
 
-class UserDict(MutableMapping): ...
-class UserList(MutableSequence): ...
-class UserString(Sequence): ...
+_UserDictT = TypeVar('_UserDictT', bound=UserDict)
+
+class UserDict(MutableMapping[_KT, _VT]):
+    def __len__(self) -> int: ...
+    def __getitem__(self, key: _KT) -> _VT: ...
+    def __setitem__(self, key: _KT, item: _VT) -> None: ...
+    def __delitem__(self, key: _KT) -> None: ...
+    def __iter__(self) -> Iterator[_KT]: ...
+    def __contains__(self, key: object) -> bool: ...
+    def copy(self: _UserDictT) -> _UserDictT: ...
+    @classmethod
+    def fromkeys(cls: Type[_UserDictT], iterable: Iterable[_KT], value: Optional[_VT] = ...) -> _UserDictT: ...
+
+_UserListT = TypeVar('_UserListT', bound=UserList)
+
+class UserList(MutableSequence[_T]):
+    def __init__(self, initlist: Optional[Iterable[_T]] = ...) -> None: ...
+    def __lt__(self, other: object) -> bool: ...
+    def __le__(self, other: object) -> bool: ...
+    def __eq__(self, other: object) -> bool: ...
+    def __gt__(self, other: object) -> bool: ...
+    def __ge__(self, other: object) -> bool: ...
```

Comment:
Could you add a comment where the comparison operators are declared abstract, necessitating the override here? I had difficulty finding that.